packages/base-styles/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+-   This package now requires Dart Sass. The legacy `sass` package is no longer supported ([#70135](https://github.com/WordPress/gutenberg/pull/70135)).
```

Comment:
I think this implication is acceptable, given that these legacy Sass compilers are both end of life.

I'll just ping @WordPress/native-mobile for a heads up, since I'm seeing some SCSS usage in native files as well. Should be fine though, I don't see any `node-sass` in our package-lock.